transaction/std/src/reserved_destination.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Object representing a destination for change from a transaction
```

Comment:
don't have a good suggestion, but looks like there is a stray use of "change" from the rename

Reply:
```suggestion
//! Object containing the public destination addresses of MobileCoin standard reserved subaddresses
```

Reply:
Indeed, addressed!